README.md

```diff
@@ -74,7 +74,10 @@ Creates a new user in your OPC installation. The user's private key will be retu
 
 Deletes the given OPC user.
 
-### knife opc user edit USERNAME
+### knife opc user edit USERNAME (options)
+
+- `-i FILENAME`, `--input FILENAME`: Will read the user information (i.e. username, email, display_name, first_name, last_name, middle_name) from the input FILENAME and update the given OPC user.
+- `-f FILENAME`, `--filename FILENAME`: Open FILENAME on $EDITOR. When finished, Updates the given OPC user.
```

Comment:
```suggestion
- `-f FILENAME`, `--filename FILENAME`: Opens FILENAME in $EDITOR. Updates the given OPC user when the file is saved.
```

Reply:
Is the update on save or close?

Reply:
@kagarmoe Update is on save and close the $Editor.